File: Baekjoon/Class1/06.2475 검증수.java
import java.io.*;
import java.util.StringTokenizer;

class Main{
    public static void main(String[] args) throws IOException {
        int input, total = 0;
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        while(st.hasMoreTokens()) {
            input = Integer.parseInt(st.nextToken());
            total += input * input;
        }
        System.out.print(total % 10);
        br.close();
    }
}

File: Baekjoon/Class1/06.2475 검증수.py
import sys
input = sys.stdin.readline

nums = map(int, input().split())
total = 0
for num in nums:
    total += num * num
print(total % 10)

'''
nums = map(int, input().split())
print(sum([num * num for num in nums]) % 10)
'''